dc-build.sh

```diff
@@ -1,5 +1,22 @@
 #/bin/bash
 
+main=`docker-compose  version  --short | cut -d '.' -f 1`
+minor=`docker-compose  version  --short | cut -d '.' -f 2`
+current=`docker-compose  version  --short`
+
+echo 'Checking docker-compose version'
+if [[ $main -lt 1 ]]; then
+  echo "$current is not supported docker-compose version, please upgrade to minimal supported version:1.28"
+  exit 1
+elif [[ $main -eq 1 ]]; then
+  if [[ $minor -lt 28 ]]; then
+    echo "$current is not supported docker-compose version, please upgrade to minimal supported version:1.28"
+    exit 1
+  fi
+fi
+
+echo 'Supported docker-compose version'
+
```

Comment:
Could this some kind of function/script that is included to avoid duplication?

Reply:
Yes, I was thinking the same. I will do it

Reply:
Done